extensions/extensions-helm/extensions-helm-spark-infrastructure/aissemble-spark-history-chart/templates/deployment.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ .Release.Name }}
+  name: {{ .Release.Name }}-shs
```

Comment:
Q: Wouldn't the release name already mention that it's a spark history server? So it'd be something like `my-spark-history-shs` which feels odd.

Reply:
The problem here is how it works when unified in a single chart in a downstream project.  When you generate out `spark-infrastructure` downstream, with a dependency on both the shs chart and sts chart, when the chart is deployed, `.Release.Name` is identical for both (specifically, `spark-infrastructure`)